tests/TestCase/DateTime/StringsTest.php

```diff
@@ -250,9 +250,9 @@ public function testToUnixString($class)
     public function toQuarterProvider()
     {
         return [
-            ['2007-12-25', 4],
-            ['2007-9-25', 3],
-            ['2007-3-25', 1],
+            ['2007-12-25', 4.0],
+            ['2007-9-25', 3.0],
+            ['2007-3-25', 1.0],
```

Comment:
I think this reveals a defect now. See documented return type of method.

Reply:
Could you make suggested change about documented return type of method?

The `toQuarterProvider` data provider seems to be the mixed type of `$expected`.

Reply:
I meant the method you are testing here :)

Reply:
Ok. Thanks and sorry for my misunderstanding.

If having any request change, please feel free to let me know :).

Reply:
I change the returned type hints for `FormattingTrait::toQuarter()` method.

Please see this latest diff on this PR :).